src/test/java/ch/wisv/areafiftylan/integration/RFIDIntegrationTest.java

```diff
@@ -48,14 +48,39 @@
     private TicketRepository ticketRepository;
 
     private RFIDLink createRfidLink(Ticket ticket) {
-        long rfid = (long) Math.floor(Math.random() * 9_000_000_000L) + 1_000_000_000L;
-        RFIDLink l = new RFIDLink(String.valueOf(rfid), ticket);
-        l = rfidLinkRepository.saveAndFlush(l);
-        return l;
+        String rfid = String.format("%010d", rfidLinkRepository.count());
+        RFIDLink link = new RFIDLink(rfid, ticket);
+        return rfidLinkRepository.saveAndFlush(link);
+    }
+
+    private RFIDLink createUnusedRfidLink(Ticket ticket) {
+        String rfid = String.format("%710d", rfidLinkRepository.count()); //pad with 7's
+        return new RFIDLink(rfid, ticket);
+    }
+
+    private RFIDLink createInvalidRfidLink(Ticket ticket) {
```

Comment:
This method isn't used, and code coverage shows we're not testing this exceptionhandling part of the controller regarding invalid rfid's